dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -28,9 +36,19 @@ RUN microdnf -y --setopt=tsflags=nodocs update && \
     microdnf install shadow-utils && \
     microdnf clean all
 {{- else }}
-RUN yum -y --setopt=tsflags=nodocs update && yum clean all
+RUN yum -y --setopt=tsflags=nodocs update \
+  {{- if (eq .BeatName "heartbeat") }}
+    && yum -y install epel-release \
+    && yum -y install atk cups gtk gdk xrandr pango.x86_64 libXcomposite.x86_64 libXcursor.x86_64 libXdamage.x86_64 \
```

Comment:
Why all these dependencies? I think some of them can be avoided (atk, cups, pango).

Reply:
These are required by chrome unfortunately, I agree it seems excessive, but I got them from an official list that I can't seem to find anymore. Testing their removal one by one is something that's worthwhile in the future, I agree